tests/com/carolinarollergirls/scoreboard/defaults/DefaultClockModelTests.java

```diff
@@ -79,149 +110,327 @@ public void test_defaults() {
 		assertEquals(ID, clock.getProviderId());
 		assertEquals(Clock.class, clock.getProviderClass());
 	}
-	
+
 	@Test
-	public void syncTimeTest() {
+	public void test_reset() {
```

Comment:
Can we be consistent in how we name tests/methods? camelCase seems to be our standard.

Reply:
Eclipse auto generates tests with `testMethodName` or `testMethodNameArgumentTypes` (when there are multiple methods with the same name). Where I have multiple tests for one method I use `testMethodName_scenario` as I find that makes it easier to see at a glance what is being tested here. I'll change the older tests in ClockModel to that format.